apache/uima-ruta
# Issue #136: Convert documentation to Asciidoc

**What's in the PR**
- Converted the documentation and moved it to a new module
- Fix issue that staging area is still prepared even if auto-staging is disabled

**How to test manually**
* Build and check if the documentation is properly included in the sources.zip
* Check the documentation itself

**Automatic testing**

**Documentation**

**Organizational**
      <sub><sup>Only dependencies under [approved licenses](http://www.apache.org/legal/resolved.html#category-a) are allowed. LICENSE and NOTICE files in the respective modules where dependencies have been added as well as in the project root have been updated.</sup></sub>
